app/src/main/java/org/p2p/wallet/swap/model/Slippage.kt

```diff
@@ -34,16 +34,17 @@ sealed class Slippage(val doubleValue: Double, val percentValue: String) : Parce
     object Five : Slippage(0.05, "5%")
 
     @Parcelize
-    data class Custom(val value: Double) : Slippage(value / PERCENT_DIVIDE_VALUE, "$value")
+    data class Custom(val value: Double) : Slippage(doubleValue = value, "${value.times(PERCENT_DIVIDE_VALUE)}%")
```

Comment:
check JupiterSwapSettingsPresenter, change custom slippage. And after resolve conflicts https://github.com/p2p-org/key-app-android/pull/1433

Reply:
already checked, 
roger that